reference/promotions.v3.yml

```
@@ -1258,6 +1264,20 @@ components:
         error:
           description: Error payload for the BigCommerce API.
           type: string
+    ErrorResponse405:
+      type: object
+      title: 405 Error Response
+      description: The requested method is not allowed for the specified resource.
+      properties:
+        status:
+          description: Method not allowed.
+          type: string
+        title:
+          description: "The error title describing the particular error."
```

seems like we can remove the quotes :) 
```suggestion
          description: The error title describing the particular error.
```